### PR TITLE
feat: add fftfreq() paddle frontend function in paddle.fft

### PR DESCRIPTION
**Pull request by p1utoze**
# PR Description 
Add fftfreq() Paddle frontend function in  `paddle.fft`

## Related Issue 

Resolves: #22948 

## Checklist 

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by p1utoze** [comment]
@D0m-inic Hi! Currently I have implemented fftfreq() function in paddle frontend but it haven't yet been implemented in the functional API. I have raised a suggestion issue #22992 for it. How should I proceed with this PR?

**Comment by p1utoze** [comment]
@D0m-inic Hi! All my test cases are passing. Can you review the code and merge it?

File: ivy/functional/frontends/paddle/fft.py [hunk 1]
```diff
@@ -16,6 +16,31 @@ def fft(x, n=None, axis=-1.0, norm="backward", name=None):
...
+        )
+    },
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def fftfreq(n, d=1.0, dtype=None, name=None):
+    if d * n == 0:
+        raise ValueError("d or n should not be 0.")
+
+    dtype = ivy.default_dtype()
```
Comment (KareemMAX) [inline ivy/functional/frontends/paddle/fft.py:35]: I think you need to check if it `None` or not before doing that

**Review by KareemMAX — CHANGES_REQUESTED** [review]
Hey @p1utoze,
Great work 🚀, just a small comment that I want you to address then we are good to run the final tests and merge

File: ivy/functional/frontends/paddle/fft.py (hunk 1 above)
Comment (p1utoze) [reply ivy/functional/frontends/paddle/fft.py]: Hey! Sorry for the late response. I have added the dtype check.

**Review by KareemMAX — APPROVED** [review]
LGTM

**Pull request merged**